test/loadTest/connections_stress_tests/connections_stress_tests.py

```diff
@@ -0,0 +1,313 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+"""
+__author__ = 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+import sys
+import requests
+import random
+import time
+import datetime
+import pymongo
+import rpyc
+import json
+import logging
+
+from requests.exceptions import ConnectionError
+
+logging.basicConfig(level=logging.DEBUG,
+                    format='%(asctime)s | %(levelname)s | %(message)s',
+                    filename='connections_stress_tests.log',
+                    filemode='w')
+# disable log messages from the Requests library
+logging.getLogger("requests").setLevel(logging.WARNING)
+
+
+class Stablished_Connections:
+    """
+    Verify that ContextBroker works properly with a large number of stablished connections
+    Tests procedure:
+    - drop the db in mongo
+    - create 5000 subscription with subject.entities.idPattern: .*
+    - verify/modify the notification listener with a delay of 10 minutes before answering.
+    - modify the ContextBroker config with: -httpTimeout 600000 -notificationMode threadpool:60000:5000 and restart it.
+    - launch an entity update, that it triggers all subscriptions.
+    - launch for a given time (in minutes) a "/version" request per second and:
+         - reports that its response is correct.
+         - reports the number of ESTABLISHED, CLOSE_WAIT and SUM connections (if `-noEstablished` param is used the established, the close_wait and the sum columns are ignored)
```

Comment:
With the current wording, it seems like SUM be a new kind of connection type, as ESTABLISHED and CLOSE_WAIT are. Thus, I'd suggest the following wording:

"reports the number of ESTABLISHED and CLOSE_WAIT connections and the sum of both (if `-noEstablished` param is used the established this information is not provided)"

In addition, maybe `-noEstablished` is now osbolete. What about using `-noConnectionInfo` instead?